src/screens/Select.js

```diff
@@ -506,20 +512,24 @@ const SelectPage = () => (
             </Description>
             <Example>"key"</Example>
           </PropertyValue>
-          <PropertyValue type="function">
+          <PropertyValue type="object">
             <Description>
-              If a function is provided, it is called with the option and should
-              return the value. If reduce is true, this value will be used for
-              the 'value' delivered via 'onChange'.
+              If reduce is true, this value will be used for the 'value'
+              delivered via 'onChange'.
             </Description>
-            <Example>
-              {`
+            <Example>{`
 {
   key: "string",
   reduce: boolean
 }
-            `}
-            </Example>
+            `}</Example>
+          </PropertyValue>
+          <PropertyValue type="function">
+            <Description>
+              If a function is provided, it is called with the option and should
+              return the value.
+            </Description>
+            <Example>{`(option) => { return "string" }`}</Example>
```

Comment:
Is "string" correct here? I'm thinking about cases where the value is a number.

Reply:
I think returning an object is also valid. I will update the example to just be `(option) => {}` since in the description it states that the function should return the value. Hopefully this is clear enough and the user can look at the documentation for the value prop to see the accepted types.